core/modules/dashboard/includes/block.news.inc

```diff
@@ -73,4 +72,53 @@ class DashboardNewsBlock extends Block {
       ),
     );
   }
+
+  /**
+   * Return the current list of news items.
+   *
+   * This response is cached. If there is no recent cache, a new one will be
+   * generated on-the-fly. If the first request fails, the cache will persist
+   * with FALSE as the saved value until the next time the cache is cleared or
+   * cron updates the news list.
+   *
+   * @return FALSE|array
+   */
+  public static function getNews() {
+    $cache = cache()->get('dashboard_news_feed');
+    // If no cache is set at all, fetch results immediately.
+    if ($cache === FALSE) {
+      $news = DashboardNewsBlock::refreshNewsFeed();
```

Comment:
Do we have to use this syntax here? It looks strange compared to the rest of the code in core. Is this the same as `$this->refreshNewsFeed();`?

Reply:
Hm. I see we are using the same `syntax in dashboard_cron()`. I might be tempted to add a procedural wrapper function if I thought this was going to be something contrib might leverage. If that turns out to be the case, we can add one later.